Szymon/Zadanie7/src/test/java/Szymon7/page/Page.java

```diff
@@ -0,0 +1,19 @@
+package Szymon7.page;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.PageFactory;
+
+public abstract class Page {
+    protected WebDriver driver;
+
+    public Page(WebDriver driver){
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+    }
+
+    public String username(int numberOfChars) {
```

Comment:
To nie jest element API testowego strony.
Taka publiczna metoda nie powinna sie tu znalezc - wszystkie publiczne metody Page powinny zwracac jakis Page (fluent interface).

Reply:
to gdzie ? :(